vagrants/archlinux_archlinux/Vagrantfile

```diff
@@ -43,6 +43,7 @@ eval "$(pyenv init -)"
     # Now that prereqs are setup, let's try the app itself!
 
     cd /app
+    make .env || true
```

Comment:
Breaking .env file creation out of `make setup` step here, and `|| true` makes it not break the vagrant setup.